core/src/test/resources/StringResolver.properties

```diff
@@ -5,4 +5,7 @@ key4=${key1}.${key3}
 key5=${key5}
 key1_value1=value101
 testMultiResolve=one,two,three_${key1}${key2},my_${key3},StageSpecifics_${key4}
-cyclic=prefix ${cyclic} suffix
\ No newline at end of file
+cyclic=prefix ${cyclic} suffix
+log.dir=c:/temp
+testdata=${log.dir}/testdata
+mfh=${testdata}/mfh
```

Comment:
Can you also add a property with a default value, so we can test that the framework can parse them without the `resolveWithPropertyName` flag enabled.